### PR TITLE
Feature/editor refactor

### PR DESCRIPTION
**Pull request by PiKatso**
This feature branch completes a refactor of the editor component, allowing us to effectively eliminate two components for edit-row/edit-grid views. This refactor emulates the previous code exactly and simply uses the html of the eliminated components, with an NgIf statement to create the same effect as the previous component routing.

### CODE REVIEW

**Comment by ccbpeden** [comment]
My only comment is that I think with the editor component not having any child components, it should not have any child routes referencing no-longer extant children. viz pages.routing.ts

     path: 'editor',
      component: EditorComponent,
      children: [
 -      {path: '', redirectTo: 'edit-row-detail', pathMatch: 'full' },
 -      {path: 'edit-row-detail', component: EditRowDetailComponent },
 -      {path: 'edit-grid-detail', component: EditGridDetailComponent },
 +      {path: '', redirectTo: 'edit-row-detail', pathMatch: 'full' }
      ],

i would just cut off the children array entirely.

**Comment by PiKatso** [comment]
fixed the requested changes. Thanks @ccbpeden

**Comment by johnbuck** [comment]
@ccbpeden @AaronSleadd 

Please review and approve/request changes first thing today. Thanks!

File: src/app/pages/editor/editor.component.scss [hunk 1]
```diff
@@ -1,5 +1,174 @@
 @import '../../theme/sass/conf/conf.scss';
 
+// GRID view scss
+.commence-content-grid {
+  margin: 0 1em 0 .8em;
+}
+
+.detail-body{
```
Comment (AaronSleadd) [inline src/app/pages/editor/editor.component.scss]: Let's rename these classes to include a reference to grid.

Comment (PiKatso) [reply src/app/pages/editor/editor.component.scss]: I replaced everything with 'detail' in the class title and from the eliminated components with either 'row-detail' or 'grid-detail' accordingly. I also identified some redundant/superfluous styling and eliminated it.

**Comment by johnbuck** [comment]
@AaronSleadd @PiKatso @Umitosan @ccbpeden Approved!!

**Pull request merged**